lib/mayaUsd/commands/Readme.md

```diff
@@ -0,0 +1,445 @@
+corresponding to an attribute (e.g. `myCustomAttr_AbcGeomScope` for
+`myCustomAttr`). Supported interpolations are `fvr` (face-varying),
+`uni` (uniform), `vtx` (vertex), and `con` (constant). If there's no
+sidecar `_AbcGeomScope`, the primvar gets exported without an authored
+interpolation; the current fallback interpolation in USD is constant
+interpolation.
+
+The type of the primvar is automatically deduced from the type of
+the Maya attribute. (We currently ignore `_AbcType` hint attributes.)
+
+Example `mayaUSDExport` invocation with `primvarprefix` option:
+
+```python
+cmds.mayaUSDExport(
+    file=usdFilePath,
+    chaser=['alembic'],
+    chaserArgs=[
+       ('alembic', 'primvarprefix', 'ABC_,ABC2_=customPrefix_,ABC3_=,ABC4_=customNamespace:'),
+    ])
+```
+
+
+## Setting Site-Specific Defaults for MayaUSDImportCommand/MayaUSDExportCommand
+
+Suppose that at your site you always want to export with the flags
+`-exportMaterialCollections` and `-chaser alembic` and `-chaserArgs ...`,
+even when exporting using the "File > Export All" menu item, where
+you wouldn't normally be able to set some more advanced options like
+`-chaser` that are only available via the specific command, for example
+the `mayaUSDExport` command.
+
+You can configure these options as the "site-specific" defaults for
+your installation of the plug-in, for example the Autodesk Maya USD
+plugins by creating a plugin and adding some information to its
+`plugInfo.json` file (see the USD Plug library documentation for more
+information).
+
+For example, your `plugInfo.json` would contain these keys if you
+wanted the default flags mentioned above:
+
+```javascript
+{
+    "UsdMaya": {
+        "mayaUSDExport": {
+            "exportMaterialCollections": true,
+            "chaser": "alembic",
+            "chaserArgs": [
+                ["alembic", "primvarprefix", "ABC_,ABC2_=customPrefix_,ABC3_=,ABC4_=customNamespace:"],
+                ["alembic", "attrprefix", "ABC_,ABC2_=customPrefix_,ABC3_=,ABC4_=customNamespace:"]
+            ]
+        }
+    }
+}
+```
+
+This also works for the `MayaUSDImportCommand` base command, for example
+in the `mayaUSDImport` command and the "File > Import" menu item; use
+the `mayaUSDImport` key in the `plugInfo.json` file to configure your
+site-specific defaults.
+
+
+## `mayaUSDListShadingModesCommand`
+
+The purpose of this command is to translate between nice-names, internal
+names and annotations for various elements passed to the other commands.
+
+### Command Flags
+
+| Long flag              | Short flag | Type           | Description |
+| ---------------------- | ---------- | -------------- | ----------- |
+| `-export`              | `-ex`      | noarg          | Retrieve the list of export shading mode nice names. |
+| `-exportOptions`       | `-eo`      | string         | Retrieve the real name of the shading mode given its nice name. These can be used in the `shadingMode` option of the export command. |
```

Comment:
Slightly more than that. Specifying a material export mode is done with two parameters:
 -shadingMode
 -convertMaterialsTo
So `exportOptions` will return the real values for both these parameters to get the expected material export.